content/efcore/getting-started/samples/loading-related-data/then-include-method-chain/Program.cs

```diff
@@ -16,11 +16,13 @@ public static void Main()
 
             foreach (Book book in author.Books)
             {
-                foreach (CheckoutRecord checkoutRecord in book.CheckoutRecords)
-                {
-                    Console.WriteLine("{0}: Due {1} - {2} {3}", book.Title, checkoutRecord.DueDate.ToString("MMMM dd, yyyy"), checkoutRecord.Reader.FirstName, checkoutRecord.Reader.LastName);
-                }
-            }
+                Console.WriteLine("Book: {0}", book.Title);
+			
+				foreach (CheckoutRecord checkoutRecord in book.CheckoutRecords)
+				{
+					Console.WriteLine("Due Date: {0}\n Checked out from Reader: {1} {2}", checkoutRecord.DueDate.ToString("MMMM dd, yyyy"), checkoutRecord.Reader.FirstName, checkoutRecord.Reader.LastName);
```

Comment:
Use multiple `Console.WriteLine` statements instead of the newlines. It will make the code more readable.